.github/workflows/main.yml

```diff
@@ -90,25 +98,32 @@ jobs:
 
       - run: yarn webpack --test --no-lavamoat --no-cache --browser=chrome --lockdown --sentry --snow --env production
```

Comment:
Why are we making a production build in a step meant to produce a test build?

Reply:
Ah, it's because this build was originally to test benchmarks on webpack, and David Murdoch recommended this build command to best replicate startup performance.  Are you saying we need to produce two `webpack --test` builds, one for `--env production` and one not?